### DISCUSSION TITLE
Error/Failure upon installing Laravel from Composer

### DISCUSSION THREAD

**Discussion by buzz-git** [laravel/framework#49415] [General]
Hello Folks,
Upon installing Laravel from Composer.exe I get the following message:

Problem 1
    - laravel/framework[v10.10.0, ..., v10.37.3] require league/flysystem ^3.8.0 -> satisfiable by league/flysystem[3.8.0, ..., 3.23.0].
    - league/flysystem[3.3.0, ..., 3.14.0] require league/mime-type-detection ^1.0.0 -> satisfiable by league/mime-type-detection[1.0.0, ..., 1.14.0].
    - league/flysystem[3.15.0, ..., 3.23.0] require league/flysystem-local ^3.0.0 -> satisfiable by league/flysystem-local[3.15.0, ..., 3.23.0].
    - league/mime-type-detection[1.0.0, ..., 1.3.0] require php ^7.2 -> your php version (8.2.13) does not satisfy that requirement.
    - league/mime-type-detection[1.4.0, ..., 1.14.0] require ext-fileinfo * -> it is missing from your system. Install or enable PHP's fileinfo extension.
    - league/flysystem-local[3.15.0, ..., 3.23.0] require ext-fileinfo * -> it is missing from your system. Install or enable PHP's fileinfo extension.
    - Root composer.json requires laravel/framework ^10.10 -> satisfiable by laravel/framework[v10.10.0, ..., v10.37.3].
    
    Any ideas what this means or why it seems my system is missing the league/flysystem/mime-type-detection. It always worked perfectly before?  Thanks for any help to resolve this.

**Comment by Dgeronivo** [comment]
Hello. I'm stuck with the same problem on win 11 and installed php on win.  Check your php.ini and uncomment `extension=fileinfo`. If you have other case, would you give more info?

**Reply by Kairat2006-dev** [reply]
Не помогло
